### DISCUSSION TITLE
垃圾框架不如Antd

### DISCUSSION THREAD

**Discussion by illaoiiiii** [tusen-ai/naive-ui#5133] [General]
弹框难用的要死，要不是公司用，狗都不用这框架

**Comment by laterdayi** [comment]
菜狗， 让你写HTML 你能写明白吗 ？

**Comment by laterdayi** [comment]
你就跟我们公司的那个菜狗前端一样，在我们组业绩最低，去哪都是业绩最低。

**Comment by 07akioni** [comment]
> 弹框难用的要死，要不是公司用，狗都不用这框架

哥，为难你了，也帮不了你啥，和你陪个不是。工作都差不多，辛苦
